I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 767, in script
    call screen drinks
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/script.rpy", line 848, in execute
    screen drinks():
  File "game/script.rpy", line 848, in execute
    screen drinks():
  File "game/script.rpy", line 849, in execute
    text "Click on the objects to drink":
  File "game/script.rpy", line 849, in keywords
    text "Click on the objects to drink":
  File "game/script.rpy", line 851, in <module>
    at sepia
NameError: name 'sepia' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 767, in script
    call screen drinks
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/ast.py", line 2259, in execute
    self.call("execute")
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/exports.py", line 3347, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 4055, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/screen.py", line 476, in visit_all
    callback(self)
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/core.py", line 4055, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/screen.py", line 487, in per_interact
    self.update()
  File "/Users/vkumar/Downloads/renpy-8.1.3-sdk/renpy/display/screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/script.rpy", line 848, in execute
    screen drinks():
  File "game/script.rpy", line 848, in execute
    screen drinks():
  File "game/script.rpy", line 849, in execute
    text "Click on the objects to drink":
  File "game/script.rpy", line 849, in keywords
    text "Click on the objects to drink":
  File "game/script.rpy", line 851, in <module>
    at sepia
NameError: name 'sepia' is not defined

macOS-13.1-arm64-arm-64bit arm64
Ren'Py 8.1.3.23091805
lovehurts 1.0
Tue Oct 17 02:30:37 2023
